src/lib/utilities/export-workflows.ts

```diff
@@ -0,0 +1,16 @@
+import type { WorkflowExecution } from '@temporalio/common';
+
+import { stringifyWithBigInt } from './parse-with-big-int';
+
+export const exportWorkflows = (workflows: WorkflowExecution[]) => {
+  const content = stringifyWithBigInt({ workflows }, null, 2);
+  download(content, `workflows-${Date.now()}.json`, 'text/plain');
+
+  function download(content: string, fileName: string, contentType: string) {
+    const a = document.createElement('a');
+    const file = new Blob([content], { type: contentType });
+    a.href = URL.createObjectURL(file);
+    a.download = fileName;
```

Comment:
Would including the range (e.g. 1-100) be helpful in the file name?

Reply:
:+1 I included the count